build.sbt

```diff
@@ -319,21 +299,13 @@ lazy val enumeratumPlay = Project(id = "enumeratum-play", base = file("enumeratu
   .settings(commonWithPublishSettings)
   .settings(testSettings)
   .settings(
-    version            := Versions.Core.head,
-    crossScalaVersions := scalaVersionsAll,
-    libraryDependencies += {
-      val dep = ("com.typesafe.play" %% "play" % "2.8.0").exclude("org.scala-lang.modules", "*")
-
-      if (scalaBinaryVersion.value == "3") {
-        dep
-          .exclude("org.scala-lang.modules", "*")
-          .exclude("com.typesafe.play", "play-json_2.13")
-          .cross(CrossVersion.for3Use2_13)
-      } else {
-        dep
-      }
-    },
-    libraryDependencies += scalaTestPlay(scalaVersion.value),
+    version := Versions.Core.head,
+    // Play do not support 2.12 (default from common settings)
+    scalaVersion                                := scala_2_13Version,
+    crossScalaVersions                          := Seq(scala_2_13Version, scala_3Version),
+    libraryDependencies += ("org.playframework" %% "play" % "3.0.0")
+      .exclude("org.scala-lang.modules", "*"),
```

Comment:
@lloydmeta It's not clear to me why the `exclude("org.scala-lang.modules", "*")` is present, I kept it in doubt but maybe it's not needed anymore?